application/data_access/odp_summaries/conformance.py

```diff
@@ -1,440 +1,442 @@
-        "headers": headers,
-        "rows": rows,
-        "stats_headers": stats_headers,
-        "stats_rows": stats_rows,
-        "params": params,
-    }, final_count[csv_out_cols]
-
-
-def make_pretty(text):
-    if type(text) is float:
-        # text is a float, make a percentage
-        return str((round(100 * text))) + "%"
-    elif "_" in text:
-        # text is a column name
-        return text.replace("_", " ").replace("pct", "%").replace("count", "")
-    return text
-
-
-def get_background_class(text):
-    if type(text) is float:
-        group = int((text * 100) / 10)
-        if group == 10:
-            return "reporting-100-background"
-        else:
-            return "reporting-" + str(group) + "0-" + str(group + 1) + "0-background"
-    return ""
-
-
-def get_dataset_field():
-    specification_df = pd.read_csv(
-        "https://raw.githubusercontent.com/digital-land/specification/main/specification/specification.csv"
-    )
-    rows = []
-    for index, row in specification_df.iterrows():
-        specification_dicts = json.loads(row["json"])
-        for dict in specification_dicts:
-            dataset = dict["dataset"]
-            fields = [field["field"] for field in dict["fields"]]
-            for field in fields:
-                rows.append({"dataset": dataset, "field": field})
-    return pd.DataFrame(rows)
+import ast
+
+import numpy as np
+import pandas as pd
+
+from application.data_access.datasette_utils import get_datasette_query
+from application.data_access.odp_summaries.utils import get_provisions
+
+SPATIAL_DATASETS = [
+    "article-4-direction-area",
+    "conservation-area",
+    "listed-building-outline",
+    "tree-preservation-zone",
+    "tree",
+]
+DOCUMENT_DATASETS = [
+    "article-4-direction",
+    "conservation-area-document",
+    "tree-preservation-order",
+]
+
+# Separate variable for all datasets as arbitrary ordering required
+ALL_DATASETS = [
+    "article-4-direction",
+    "article-4-direction-area",
+    "conservation-area",
+    "conservation-area-document",
+    "listed-building-outline",
+    "tree-preservation-order",
+    "tree-preservation-zone",
+    "tree",
+]
+
+# Configs that are passed to the front end for the filters
+DATASET_TYPES = [
+    {"name": "Spatial", "id": "spatial"},
+    {"name": "Document", "id": "document"},
+]
+
+COHORTS = [
+    {"name": "RIPA Beta", "id": "RIPA-Beta"},
+    {"name": "RIPA BOPS", "id": "RIPA-BOPS"},
+    {"name": "ODP Track 1", "id": "ODP-Track1"},
+    {"name": "ODP Track 2", "id": "ODP-Track2"},
+    {"name": "ODP Track 3", "id": "ODP-Track3"},
+    {"name": "ODP Track 4", "id": "ODP-Track4"},
+]
+
+
+def get_column_field_summary(dataset_clause, offset):
+    sql = f"""
+    SELECT edrs.*, rle.licence
+    FROM endpoint_dataset_resource_summary AS edrs
+    LEFT JOIN (
+        SELECT endpoint, licence
+        FROM reporting_latest_endpoints
+    ) AS rle ON edrs.endpoint = rle.endpoint
+    WHERE edrs.resource != ''
```

Comment:
you've mistakenly removed an `and endpoint_end_date=''` clause in this query. 
we aren't ready to remove that clause yet (we need the endpoint_dataset_summary table to exist first)